<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TakePictureClass</class>
 <widget class="QWidget" name="TakePictureClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TakePicture</string>
  </property>
  <widget class="QPushButton" name="OpenCameraButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>420</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>打开相机</string>
   </property>
  </widget>
  <widget class="QPushButton" name="TakePictureButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>440</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>拍照</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CloseCameraButton">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>450</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭相机</string>
   </property>
  </widget>
  <widget class="QLabel" name="pictureLabel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>70</y>
     <width>391</width>
     <height>271</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>390</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>曝光时间</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ExposureTimeEdit">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>390</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>390</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>us</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CalibCamButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>390</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>相机标定</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="TakePicture.qrc"/>
 </resources>
 <connections/>
</ui>
